src/audio/tdfb/tdfb.c

```diff
@@ -441,6 +782,11 @@ static int tdfb_copy(struct comp_dev *dev)
 			     n * cl.sink_frame_bytes);
 	}
 
+	/* TODO: Update direction of arrival estimate */
```

Comment:
??

Reply:
This TODO can be removed, it's same as above. Copy() like this should be final.